bokehjs/src/less/pretty.less

```diff
@@ -1,3 +1,6 @@
+// @mattpap - these colors should probably be parameterized
+// but I think this is pretty much only used by the examiner,
+// so maybe not high enough priority to fix?
 .bk-nullish { color: #7724c1; }
 .bk-token { color: #881280; }
 .bk-boolean { color: #007500; }
```

Comment:
You can leave them out for now. These are specific to pretty printing models, properties and values, used primarily in examiner component for the time being.